examples/resources/xray_security_policy/resource.tf

```diff
@@ -0,0 +1,56 @@
+resource "xray_security_policy" "min_severity" {
+  name        = "test-security-policy-severity"
+  description = "Security policy description"
+  type        = "security"
+  rules {
+    name     = "rule-name-severity"
+    priority = 1
+    criteria {
+      min_severity = "High"
+    }
+    actions {
```

Comment:
Ok, @alexhung  and @danielmkn  - in the projects repo it's repeated singular (so, the analog here would be 
```hcl
action {
}
action{
}
``` 

and here it's `actions`. Using plural, I would expect it to be an array `[]` - I am pretty sure it's supposed to be repeated, singular. We need to be consistent and, if possible, in line with the rest of the community

Reply:
I don't agree with a singular form `action` here. This is the list (even if it's a TypeSet) of actions in the policy, but we only can have one set of actions per the rule. For the `rules` I agree, we should use a singular form because we can add multiple rules per one policy.